docs/source/development/defining_new_models.md

```diff
@@ -0,0 +1,221 @@
+# How to add a new model to the Virtual Rainforest
+
+The Virtual Rainforest is designed to be modular, meaning that the set of models to be
+used in a particular run is configurable at the start of the simulation. We are starting
+out by defining a core set of models (`abiotic`, `animals`, `plants` and `soil`), which
+will generally all be included for the vast majority of simulations. In future, it might
+be desirable to include models for other aspects of rainforests (e.g. `freshwater`), or
+to include multiple modelling approaches for a process. When this happens a new model
+should be created. This page will set out the process for adding a new model to the
+Virtual Rainforest in a manner that allows it to be used appropriately by the `core`
+simulation functionality.
+
+## Define a new `Model` class
+
+You should first start by defining a new folder for your model (within
+`virtual_rainforest/models/`).
+
+```bash
+mkdir virtual_rainforest/models/freshwater
+```
+
+Within this folder a `python` script defining the model should be created.
+
+```bash
+touch virtual_rainforest/models/freshwater/model.py
+```
+
+This script must import the [`BaseModel` class](../api/core/model.md) as new class must
```

Comment:
Just to mention I am aware that this link is broken, it links to a page that exists in the `feature/data_system` branch but not in `develop`, so once that is merged the link won't be broken any longer.